aacgmv2/deprecated.py

```diff
@@ -67,7 +67,6 @@ def convert(lat, lon, alt, date=None, a2g=False, trace=False, allowtrace=False,
         estr += ' must either use field-line tracing (trace=True '
         estr += 'or allowtrace=True) or indicate you know this is a bad idea'
```

Comment:
It's the maximum height for using the coefficients.  I didn't add an in-code comment because it's explained in the error message.  Do you think it would be better to have a in-code comment as well?

Reply:
Nah, I think that should be fine.